plugin/.ktlint.yml

```
@@ -0,0 +1,46 @@
+
+# Line length
+max_line_length: 100
+
+# Allow wildcard imports for convenience
+disabled_rules:
+  - no-wildcard-imports
+
+# Standard formatting rules
+ktlint_standard_chain-wrapping:
+  disabled: false
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)